ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -30,5 +30,176 @@
 #include <catch2/catch.hpp>
 
 #include "azure_connection.h"
+#include <fstream>
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+static std::string
+create_file(const std::string &object_name, const std::string &payload)
+{
+    std::ofstream file(object_name + ".txt");
+    file << payload;
+    file.close();
+    return object_name + ".txt";
+}
+
+static std::string obj_prefix("azuretest/unit/"); // To be concatenated with a random string.
+
+// Concatenates a random suffix to the prefix being used for the test object keys. Example of
+// generated test prefix: "azuretest/unit/2023-31-01-16-34-10/623843294--".
+static int
+randomize_test_prefix()
+{
+    char time_str[100];
+    std::time_t t = std::time(nullptr);
+
+    REQUIRE(std::strftime(time_str, sizeof(time_str), "%F-%H-%M-%S", std::localtime(&t)) != 0);
+
+    obj_prefix += time_str;
+
+    // Create a random device and use it to generate a random seed to initialize the generator.
+    std::random_device my_random_device;
+    unsigned seed = my_random_device();
+    std::default_random_engine my_random_engine(seed);
+
+    obj_prefix += '/' + std::to_string(my_random_engine());
+    obj_prefix += "--";
+
+    return 0;
+}
+
+TEST_CASE("Testing Azure Connection Class", "azure-connection")
+{
+    auto azure_client = Azure::Storage::Blobs::BlobContainerClient::CreateFromConnectionString(
+      std::getenv("AZURE_STORAGE_CONNECTION_STRING"), "myblobcontainer1");
+    bool exists = false;
+
+    REQUIRE(randomize_test_prefix() == 0);
```

Comment:
You might have discussed this already, but I think it would be cleaner to have `randomize_test_prefix` return the prefix. Then you don't need the global thing and it's a little clearer what happens when they all run in parallel.

Reply:
makes sense i've removed the global var and made `randomize_test_prefix` return the prefix

Reply:
https://github.com/wiredtiger/wiredtiger/pull/8747/commits/27d5d3c123e9a19501ffdc1d27fccbadc121470e